managed/services/agents/agents.go

```diff
@@ -138,22 +138,23 @@ func ensureAuthParams(exporter *models.Agent, params *agentv1.SetStateRequest_Ag
 ) error {
 	if agentVersion.Less(minAuthVersion) {
 		params.Env = append(params.Env, fmt.Sprintf("HTTP_AUTH=pmm:%s", exporter.GetAgentPassword()))
+		return nil
+	}
+
+	if params.TextFiles == nil {
+		params.TextFiles = make(map[string]string)
+	}
+
+	wcf, err := exporter.BuildWebConfigFile()
+	if err != nil {
+		return err
+	}
+	params.TextFiles["webConfigPlaceholder"] = wcf
```

Comment:
```suggestion
	params.TextFiles["webConfig"] = wcf
```

Reply:
Are we sure that this will not disrupt the upgrades?

Reply:
it shouldn't as we send everything from server and don't have it on client side.

Reply:
Done.